src/common/kvcache_mgr.h

```diff
@@ -28,6 +28,7 @@ class KVCacheMgrImplBase {
     virtual bool reorderCache(const std::vector<int> &seqIDs, const std::vector<int> &prevSeqIDs) = 0;
     virtual bool addPrefix(int prefixId, int seqID) = 0;
     virtual bool prepareCache(const std::vector<int> &seqIDs) = 0;
+    virtual bool isExist(int seqID) const = 0;
```

Comment:
is and exist are both verb. Just 'exist' may be OK.

Reply:
done